src/Components/NavBar.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import {
+  HiOutlineHeart,
+  HiOutlineShoppingCart,
+  HiSearch,
+  HiOutlineUserCircle
+} from "react-icons/hi";
+import { Link } from "react-router-dom";
+
+import { useData } from "../Contexts/dataProvider";
+import Badge from "./Badge";
+import { useAuth } from "../Contexts/authProvider";
+
+export function NavBar() {
+  const {
+    state: { cartItems, wishListItems },
+    dispatch
+  } = useData();
+  const { login, logoutUser } = useAuth();
+  const [itemToSearch, setItemToSearch] = useState("");
+  const TotalItemsInCart = cartItems.reduce(
+    (count, item) => count + item.quantity,
+    0
+  );
+  function searchItems() {
+    dispatch({ type: "SEARCH_ITEM", payload: itemToSearch });
+    setItemToSearch("");
+  }
+
+  return (
+    <nav>
+      <div className="activeNav flex-row flex-gap">
+        <Link className="nav-header link-btn" to="/">
```

Comment:
Hey,
Instead of using Link, you could look into Navlinks, which is more preferrable in case of navbars.
One of the advantages of navlinks would be you could in future display the active link by changing the styles.

I have added the source to it as well

https://reactrouter.com/web/api/NavLink

Reply:
Yeah thanks for suggesting this, I have made the changes.